plz.el

```diff
@@ -449,11 +449,14 @@ NOQUERY is passed to `make-process', which see.
                       (progn
                         (write-region (point-min) (point-max) filename)
                         (funcall then filename))
+                    (file-already-exists
+                     (funcall then (make-plz-error :message (format "error while writing to file %S: %S" filename err))))
```

Comment:
Should we be calling the `then` or `else` function?

Reply:
What would the alternative be?

Reply:
I mean, IIUC we should be calling `else`, not `then`, right?

Reply:
I'm not sure.

1. The pattern `(funcall then (make-plz-error ...))` shows up several other times in the function.
2. If we do want to call `else`, then I think we need to ensure that `else` is bound, since it can be `nil`.

Reply:
Sorry, I should have looked more closely at the code before asking about this.  You're right, `then` is the one to call here.